receiver/receivercreator/README.md

```diff
@@ -486,6 +486,15 @@ receiver_creator/metrics:
      enabled: true
      # Define which receivers should be ignored when provided through annotations
      # ignore_receivers: []
+receiver_creator/logs:
+  watch_observers: [ k8s_observer ]
+  discovery:
+    enabled: true
+     # Enable log collection for all discovered Pods/containers unless they are explicitly
+     # annotated with `io.opentelemetry.discovery.logs/enabled: "false"`. To avoid collecting Collector's
+     # own logs make sure that Collector Pods are properly annotated
+     # with `io.opentelemetry.discovery.logs/enabled: "false"`
+     # default_log_collection: true
```

Comment:
If I understand correctly, the setting `default_log_collection: true` is equivalent to having the following annotation on every pod: `io.opentelemetry.discovery.logs/enabled: true`.

Why not extend this possibility to being able to set a default for any annotation? Like this:


```yaml
receiver_creator/metrics:
  watch_observers: [ k8s_observer ]
  discovery:
    enabled: true
    default_annotations:
      io.opentelemetry.discovery.logs/enabled: true
```

This way users could also specify other default settings, like a default File Log receiver configuration, or a default metrics scraper and its config (but maybe not enabling it by default), etc.

Reply:
I love this idea! Thank's for the suggestion! Pushed this in a different commit.